src/components/CytoscapeGraph/CytoscapeGraphUtils.ts

```diff
@@ -67,14 +67,15 @@ export const CyNode = {
 };
 
 export const ZoomOptions = {
-  fitPadding: 25
+  fitPadding: 25,
+  maxZoom: 2.5
 };
 
 export const safeFit = (cy: Cy.Core, centerElements?: Cy.Collection) => {
   cy.fit(centerElements, ZoomOptions.fitPadding);
-  if (cy.zoom() > 2.5) {
-    cy.zoom(2.5);
-    cy.center(centerElements);
+  if (cy.zoom() > ZoomOptions.maxZoom) {
+    cy.zoom(ZoomOptions.maxZoom);
+    !!centerElements && !!centerElements.length ? cy.center(centerElements) : cy.center();
```

Comment:
I think double bangs are not that elegant nowadays. There is a *Reasons for using double negation* and a *reasons agains using double negation* in [this sitepont article](https://www.sitepoint.com/javascript-double-negation-trick-trouble/). IMO, it's usage is a personal preference.

My preference is to use double negation only if a boolean value is strictly needed, instead of a truthy/falsy value. Here I'd prefer `centerElements && centerElements.length != 0 ? cy.center(centerElements) : cy.center();`. 

Again, it's a personal preference. If this is your preferred way, I'm fine :)

Reply:
Maybe it is uglier but I like that it is explicit about being a boolean test for whatever the datatype may be.  I guess I'm subjectively for it :)

Reply:
Any code style is subjective :)